src/models/Membership.js

```diff
@@ -0,0 +1,16 @@
+export default function(sequelize, DataTypes) {
+    var Membership = sequelize.define('Membership', {
+        membershiptype: {
+            type: DataTypes.ENUM('GOLD', 'PLATINUM', 'DIAMOND'),
+        },
+        createdBy: DataTypes.INTEGER,
+        updatedBy: DataTypes.INTEGER
+    }, {
+        associate: (models) => {
+            Membership.hasMany(models.Profile, { foreignKey: 'membership_id' });
```

Comment:
profile is related to membership, but not reverse.

user  profile can have member ship like gold, silver, etc 
thats all